packages/core/client/src/schema-component/antd/icon-picker/__tests__/icon-picker.test.tsx

```diff
@@ -47,4 +47,15 @@ describe('IconPicker', () => {
     await userEvent.click(icons[1]);
     expect(screen.queryAllByRole('img').length).toBe(0);
   }, 300000);
+
+  it('should filter the displayed icons when changing the value of search input', async () => {
+    const { container } = render(<App />);
+
+    const button = container.querySelector('button') as HTMLButtonElement;
+    await userEvent.click(button);
+
+    const searchInput = container.querySelector('input[name="icon-search"]') as HTMLInputElement;
+    fireEvent.change(searchInput, { target: { value: 'left' } });
```

Comment:
Is it better to use `await userEvent.type()` instead of `fireEvent.change`?

Reply:
See [this](https://github.com/nocobase/nocobase/pull/4609#discussion_r1641523670)